src/libappimage/utils/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_library(
+    appimage_utils OBJECT
+    MagicBytesChecker.cpp
+    ElfFile.cpp
+    HashLib.cpp
```

Comment:
This should be a static library, not an `OBJECT` one.

Reply:
It's not part of the public interface. Do you mind to explain what uses do you have in mind for it?